### PR TITLE
correct experimental information in docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Some experimental information in the documentation is not correct. This PR corrects the information.

- Statistics V2 has been GA is v5.4.
- Cascading Placement Rules has been GA in v5.0.
- View has been GA in v3.0.
- SHOW CONFIG has been GA in v4.0.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ This document lists the features supported in each TiDB version. Note that suppo
 | [Auto increment](/auto-increment.md)                         | Y | Y  | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
 | [Auto random](/auto-random.md)                               | Y | Y  | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
 | [DDL algorithm assertions](/sql-statements/sql-statement-alter-table.md) | Y | Y  | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
-| Multi schema change: add column(s)                           | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Multi-schema change: add column(s)](/system-variables.md#tidb_enable_change_multi_schema)                           | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
```
Comment (TomShawn) [inline basic-features.md:95]: This feature has been GA in v6.2.

File: experimental-features.md [hunk 2]
```diff
@@ -18,27 +18,20 @@ This document introduces the experimental features of TiDB in different versions
 
 + Improve the stability of the optimizer's choice of indexes (Introduced in v5.0)
     + Extend the statistics feature by collecting the multi-column order dependency information.
```
Comment (TomShawn) [inline experimental-features.md:20]:
```suggestion
+ Improve the stability of the optimizer's choice of indexes: extend the statistics feature by collecting the multi-column order dependency information (Introduced in v5.0).
```

File: experimental-features.md [hunk 3]
```diff
@@ -16,29 +16,21 @@ This document introduces the experimental features of TiDB in different versions
...
++ Improve the stability of the optimizer's choice of indexes: extend the statistics feature by collecting the multi-column order dependency information (Introduced in v5.0).
 + When TiKV is deployed with limited resources, if the foreground of TiKV processes too many read and write requests, the CPU resources used by the background are occupied to help process such requests, which affects the performance stability of TiKV. To avoid this situation, you can use the [Quota Limiter](/tikv-configuration-file.md#quota) to limit the CPU resources to be used by the foreground. (Introduced in v6.0)
 
 ## Scheduling
 
-+ Cascading Placement Rules feature. It is a replica rule system that guides PD to generate corresponding schedules for different types of data. By combining different scheduling rules, you can finely control the attributes of any continuous data range, such as the number of replicas, the storage location, the host type, whether to participate in Raft election, and whether to act as the Raft leader. See [Cascading Placement Rules](/configure-placement-rules.md) for details. (Introduced in v4.0)
```
Comment (TomShawn) [inline experimental-features.md:26]: GA in v5.3

File: experimental-features.md [hunk 4]
```diff
@@ -16,29 +16,21 @@ This document introduces the experimental features of TiDB in different versions
 
 ## Stability
 
-+ Improve the stability of the optimizer's choice of indexes (Introduced in v5.0)
-    + Extend the statistics feature by collecting the multi-column order dependency information.
-    + Refactor the statistics module, including deleting the `TopN` value from `CMSKetch` and the histogram, and adding NDV information for histogram buckets of each table index. For details, see descriptions about [Statistics - `tidb_analyze_version = 2`](/statistics.md).
```
Comment (TomShawn) [inline experimental-features.md:21]: GA in v5.4

**Review by lilin90 — APPROVED** [review]
LGTM

File: experimental-features.md [hunk 5]
```diff
@@ -16,29 +16,21 @@ This document introduces the experimental features of TiDB in different versions
...
+Elastic scheduling feature. It enables the TiDB cluster to dynamically scale out and in on Kubernetes based on real-time workloads, which effectively reduces the stress during your application's peak hours and saves overheads. See [Enable TidbCluster Auto-scaling](https://docs.pingcap.com/tidb-in-kubernetes/stable/enable-tidb-cluster-auto-scaling) for details. (Introduced in v4.0)
 
 ## SQL
 
 + The expression index feature. The expression index is also called the function-based index. When you create an index, the index fields do not have to be a specific column but can be an expression calculated from one or more columns. This feature is useful for quickly accessing the calculation-based tables. See [Expression index](/sql-statements/sql-statement-create-index.md) for details. (Introduced in v4.0)
 + [Generated Columns](/generated-columns.md) (Introduced in v2.1)
 + [User-Defined Variables](/user-defined-variables.md) (Introduced in v2.1)
 + [JSON data type](/data-type-json.md) and [JSON functions](/functions-and-operators/json-functions.md) (Introduced in v2.1)
-+ [View](/information-schema/information-schema-views.md) (Introduced in v2.1)
```
Comment (TomShawn) [inline experimental-features.md:35]: GA in 3.0

File: experimental-features.md [hunk 6]
```diff
@@ -16,29 +16,21 @@ This document introduces the experimental features of TiDB in different versions
...
 + [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md) (Introduced in v6.1.0)
 
-## Configuration management
-
-+ [SHOW CONFIG](/sql-statements/sql-statement-show-config.md) (Introduced in v4.0)
```
Comment (TomShawn) [inline experimental-features.md:40]: GA in 4.0

File: statistics.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs/dev/statistics/','/docs/dev/reference/performance/statistics/']
 
 # Introduction to Statistics
 
-TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. In versions before v5.1.0, the default value of this variable is `1`. In v5.3.0 and later versions, the default value of this variable is `2`, which serves as an experimental feature. If your cluster is upgraded from a version earlier than v5.3.0 to v5.3.0 or later, the default value of `tidb_analyze_version` does not change.
```
Comment (TomShawn) [inline statistics.md:9]: GA in 5.4

File: placement-rules-in-sql.md [hunk 8]
```diff
@@ -258,7 +258,7 @@ In dictionary format, constraints also indicate a number of instances that apply
 
 ## Known limitations
 
-The following known limitations exist in the experimental release of Placement Rules in SQL:
+The following known limitations are as follows:
```
Comment (TomShawn) [inline placement-rules-in-sql.md:261]: GA in 5.4

**Comment by easonn7** [comment]
LGTM

File: basic-features.md [hunk 9]
```diff
@@ -175,7 +175,7 @@ This document lists the features supported in each TiDB version. Note that suppo
 | [Built-in physical backup](/br/backup-and-restore-use-cases.md) | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
 | [Global Kill](/sql-statements/sql-statement-kill.md)       | Y | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
 | [Lock View](/information-schema/information-schema-data-lock-waits.md) | Y | Y | Y            |      Y       |      Y       | Experimental | Experimental | Experimental |
-| [`SHOW CONFIG`](/sql-statements/sql-statement-show-config.md) | Experimental | Experimental| Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [`SHOW CONFIG`](/sql-statements/sql-statement-show-config.md) | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (TomShawn) [inline basic-features.md:178]: GA in v4.0

**Pull request merged**